docs/gettinginvolved/infrastructure/packaging.rst

```diff
@@ -60,3 +60,44 @@ The commands above require the ``rpmdevtools`` and ``yum-utils`` packages instal
 Flocker depends on a number of packages which aren't available in fedora.
 These packages are available from `our Copr repository <https://copr.fedoraproject.org/coprs/tomprince/hybridlogic/>`_.
 To enable yum to find them, put the `repo file <https://copr.fedoraproject.org/coprs/tomprince/hybridlogic/repo/fedora-20-x86_64/tomprince-hybridlogic-fedora-20-x86_64.repo>`_ in :file:`/etc/yum.repos.d/`.
+
+
+Package Hosting
+===============
+
+Some packages are hosted on Google Cloud Storage and others are hosted on Amazon S3.
```

Comment:
Below this there is a mention that OS X uses GCS. Later there is a "Legacy" section.